app/uk/gov/hmrc/customs/emailfrontend/model/EmailDetails.scala

```diff
@@ -16,24 +16,40 @@
 
 package uk.gov.hmrc.customs.emailfrontend.model
 
-import org.joda.time.DateTime
-import play.api.libs.json.{JsValue, Json}
+import play.api.libs.json.{Format, JsError, JsResult, JsSuccess, JsValue, Json, OFormat}
 
-case class EmailDetails(currentEmail: Option[String], newEmail: String, timestamp: Option[DateTime]) {
+import java.time.format.DateTimeFormatter
+import java.time.{LocalDateTime, ZoneOffset}
+
+case class EmailDetails(currentEmail: Option[String], newEmail: String, timestamp: Option[LocalDateTime]) {
 
   private val twoHours = 2
 
   lazy val amendmentInProgress = timestamp match {
-    case Some(date) => !date.isBefore(DateTime.now.minusHours(twoHours))
+    case Some(date) => !date.isBefore(LocalDateTime.now.atOffset(ZoneOffset.UTC).minusHours(twoHours).toLocalDateTime)
     case None => false
   }
 }
 
 object EmailDetails {
 
-  import uk.gov.hmrc.customs.emailfrontend.DateTimeUtil._
+  implicit val localDateTimeFormat: Format[LocalDateTime] = new Format[LocalDateTime] {
+    private val formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'")
```

Comment:
Could you please move this formatter in Utils as it is being used at lot of places and make use of the same at all the places? Please keep the meaningful name for the val that should suggest the datetime format